lib/pronto/pronto_simplecov/runner.rb

```diff
@@ -12,7 +12,7 @@ def process(patch)
         file_coverage = coverage[patch.new_file_full_path.to_s]
         return unless file_coverage
         patch.added_lines
-             .select { |line| file_coverage.line(line.new_lineno).missed? }
+             .select { |line| file_coverage.line(line.new_lineno)&.missed? }
```

Comment:
Since ruby 2.2 is not EOL yet I think it would be nice to keep support. Calling `file_coverage` twice should not impact the performance much since it is just a hash lookup.